common/params_pyx.pyx

```diff
@@ -16,6 +16,7 @@ cdef enum TxType:
 keys = {
   b"AccessToken": [TxType.CLEAR_ON_MANAGER_START],
   b"AthenadPid": [TxType.PERSISTENT],
+  b"CachedDriveStats": [TxType.PERSISTENT],
```

Comment:
can we make one param called something like api cache and use it for all of them?

Reply:
A json file containing all the results of different api calls?